modules/installation-gcp-permissions.adoc

```diff
@@ -40,6 +309,35 @@ ifdef::template[]
 .Required roles for user-provisioned GCP infrastructure
 * Deployment Manager Editor
 * Service Account Key Admin
+
+The following are the additional permissions required for user-provisioned GCP infrastructure for creating and deleting the {product-title} cluster.
+
+.Required Images permissions
+[%collapsible]
+====
+* `compute.images.create`
+* `compute.images.delete`
+* `compute.images.get`
+* `compute.images.list`
+====
+
+.Required permissions to get Region related information
+[%collapsible]
+====
+* `compute.regions.get`
+====
+
+.Required deployment manager permissions
```

Comment:
@xenolinux Any specific reason you changed the  ``Deployment Manager`` to ``deployment manager``?

Reply:
Deployment Manager is a GCP specific term. The existing documentation of UPI installation for GCP has used the term ``Deployment Manager`` and not ``deployment manager``. https://docs.openshift.com/container-platform/4.12/installing/installing_gcp/installing-gcp-user-infra.html

Reply:
I referred to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references and https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc and thought it should be in small letters. But yes you are right. If that's the case then we should keep it in caps

Reply:
Reverted the changes - Addressed